packages/shared/src/utils/currency/currency.ts

```diff
@@ -65,53 +93,54 @@ export const formatMoney = (currency_value, amount, exclude_currency, decimals =
     return sign + (exclude_currency ? '' : formatCurrency(currency_value)) + money;
 };
 
-export const formatCurrency = currency => {
+export const formatCurrency = (currency: string) => {
     return `<span class="symbols ${(currency || '').toLowerCase()}"></span>`;
 };
 
-export const addComma = (num, decimal_points, is_crypto) => {
-    let number = String(num || 0).replace(/,/g, '');
+export const addComma = (num?: number | string | null, decimal_points?: number, is_crypto?: boolean) => {
+    let number: number | string = String(num || 0).replace(/,/g, '');
```

Comment:
I can't just use type `number` here, because we are changing it to a string

Reply:
@niloo-fs Yes, I understand that but that's the whole point of TS 😅 We should fix the logic and not reassign the variable with a different type instead of ignoring the TS error 😬

Reply:
got it let me see how can I change that sweat_smile